tests/sklearn/test_sklearn_models.py

```diff
@@ -424,6 +437,10 @@ def check_serialization_dump_load(model, x, use_dump_method):
         y_pred_loaded_sklearn_model = loaded_model.sklearn_model.predict(x)
         assert numpy.array_equal(y_pred_sklearn_model, y_pred_loaded_sklearn_model)
 
+        # Check if the graphs are identical
```

Comment:
@jfrery, here is the graph check.

The fhe_circuit is not serialized, so i had to compile le loaded_model
Is there another better way to do it ?